Podfile.lock

```diff
@@ -69,6 +70,7 @@ DEPENDENCIES:
   - SocketRocket (from `https://github.com/square/SocketRocket.git`, commit `954750c018`)
   - SSKeychain
   - TwistedOakCollapsingFutures (~> 1.0)
+  - UICKeyChainStore (from `Podspecs/UICKeyChainStore.podspec`)
```

Comment:
No, I wouldn't do that. You can access the keychain items with SSKeychain APIs.
The syntax of the API call is similar.